source/features/preview-hidden-comments.tsx

```diff
@@ -0,0 +1,39 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+const REASON_MAP = {
+	resolved: 'Resolved',
+	outdated: 'Outdated',
+	'off-topic': 'Off Topic',
+	'disruptive content': 'Abuse',
+	spam: 'Spam'
+};
+
+const reasonRegExp = /^This comment was marked as ([^.]+)\.$/;
+const parseMinimizationReason = (header: HTMLElement): string => {
+	const [, reason]: any[] = header.textContent.trim().match(reasonRegExp) || [];
+	return REASON_MAP[reason] || 'Minimized';
+};
+
+const init = () => {
+	for (const details of select.all('.timeline-comment-group > .minimized-comment:not(.d-none) > details:not(.rgh-preview-hidden-comments)')) {
```

Comment:
Does this work also on comments in Commits and Files tab and review comments?

Reply:
Not to sure if I get the question.

The `selector` only selects "timeline" comments so only the user comments in the Issue or PR Conversation pages. *(The comments that get added at the end of the page)*

If the question is if this could also work for "resolved" inline comments then not with the current implementation.
Inline comments are more complex e.g. we would need to take the first comment and there could be something like "Suggested change" in there.
But I guess it could be possible to do something like this.

Reply:
I mean this comment:

<img width="667" alt="screenshot 2019-02-09 at 10 19 42" src="https://user-images.githubusercontent.com/1402241/52515274-3f286e00-2c54-11e9-8fb8-d6a8421c54bd.png">

It should be possible with minimal selector adjustment

Reply:
Also available on the files tab:

https://github.com/sindresorhus/refined-github/pull/1755/files/794b11f52d7e8e7721d3f4918ae55eb9d203b208#r254981750

And commits tab:

https://github.com/sindresorhus/refined-github/pull/1755/commits/794b11f52d7e8e7721d3f4918ae55eb9d203b208#r254981750

Reply:
Didn't realize that was possible. :smile:
Will look into it.

Reply:
> And commits tab:
> 
> [794b11f#r254981750](https://github.com/sindresorhus/refined-github/commit/794b11f52d7e8e7721d3f4918ae55eb9d203b208#r254981750)

Don't see the comments there, but I made it work. :smile:

Reply:
Also fixed some styling for linked hidden comments, seen here
> Also available on the files tab:
> 
> [#1755/files/`794b11f`#r254981750](https://github.com/sindresorhus/refined-github/pull/1755/files/794b11f52d7e8e7721d3f4918ae55eb9d203b208#r254981750)

![image](https://user-images.githubusercontent.com/9264728/52519914-296e7500-2c63-11e9-9468-54cd679107d4.png)